src/appleseed/foundation/image/colorspace.h

```diff
@@ -236,10 +237,19 @@ Color<T, 3> linear_rgb_to_hsl(const Color<T, 3>& linear_rgb);
 template <typename T>
 Color<T, 3> ciexyz_to_linear_rgb(const Color<T, 3>& xyz);
 
+template <typename T>
+Color<T, 3> ciexyz_to_linear_rgb(
```

Comment:
appleseed/foundation/image is meant to have a wider use than just in appleseed the renderer, that's why it contains a bunch of functions that are not (or no longer) used in appleseed, e.g. those dealing with sRGB. Since it has to deal with various RGB flavors, I'd prefer that we are always explicit about which flavor: linear RGB, sRGB, R'G'B'. I'm in favor of introducing `rgb_prime` where necessary.